docs/src/modules/components/DemoPropsForm.tsx

```diff
@@ -129,15 +129,15 @@ function ControlledColorRadio(props: any) {
           <Check
             fontSize="medium"
             color="inherit"
-            sx={{
+            sx={(theme) => ({
```

Comment:
Is this for future Pigment CSS support?
We seem to miss this in the migration guide https://next.mui.com/material-ui/migration/migrating-to-pigment-css/

Reply:
AFAIK it's not included in the migration guide as it's covered by the codemod.

@siriwatknp, what do you think about adding an example for all cases covered by the codemod? Would there be too many?

Reply:
For small cases like this, I can update the migration page but it should not block the v6.

Reply:
Let's create an issue to add the additional examples, but I agree not to block the stable release on this one

Reply:
Sure, I can update the codemod to link to the migration as well if that helps.